src/components/internal/PersonalDetails/types.ts

```diff
@@ -9,14 +10,15 @@ export interface PersonalDetailsProps {
     onChange: (newState: object) => void;
     readonly?: boolean;
     ref?: any;
+    validator?: ValidatorI;
 }
 
 export interface PersonalDetailsStateError {
     firstName?: boolean;
     lastName?: boolean;
     gender?: boolean;
-    dateOfBirth?: boolean;
-    telephoneNumber?: boolean;
+    dateOfBirth?: string;
+    telephoneNumber?: string;
```

Comment:
This could be `string | bolean` for cases where we don't want to show an error message.

Reply:
Fixed